src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
     public static void main(String[] args) {
```

Comment:
The `main` method should not be static if we are to avoid using static methods according to the checklist. Consider creating an instance of `Application` and then calling a non-static method to execute the logic.

Reply:
Ignore this